### ISSUE TITLE
Feature Request: Manually Execute (for DM) on Click

### ISSUE THREAD

**Issue by davidhawbecker** [ironmonk88/monks-active-tiles#108]
Dude, first I must say your stuff is epic and a literal game changer, so thank you for all you do. I don't know if this has already been brought up, but I've noticed I tend to use this module a lot from a Manual trigger stand point. i.e. Players walk into the area and I select the tile tool, right click on the active tile and click the manual power button. This is obviously a multi-step process to manually activate the tile and during a game as I'm sure you know anything to stream line the workflow is hugely appreciated. My question is whether there's any way to set this up as a single step OR two step process at the most? Maybe a macro that manually executes a selected tile OR a click to activate (for the DM). 

TL;DR: Can you create an " Click to Execute"  for the DM to manually execute an active tile? 

If this is outside the realm of possibility, no worries, I still love all your stuff. Thank you again.

**Comment by davidhawbecker** [comment]
Disregard. I made his way more complicated than it needed to be.

**Comment by ironmonk88** [comment]
That happens.  :-) Just wanted to confirm that you can achieve this by setting it to On Click, and then limiting it to being Controlled by `GM Only`.

And also if you wanted to join my Discord server.  https://discord.gg/m4NRPTRU
There's a really great community on there that has a lot of examples and ideas for Active Tiles.

**Comment by davidhawbecker** [comment]
Totally. That's exactly what I figured out. 

I was looking for a way to join your server earlier so I guess my plan worked. :) Seriously man, thank you. This stuff creates so many awesome ways to run a game that never existed before. You're putting out some amazing stuff.
